son/content/intermediate_outcomes/work_in_early_adulthood_(25_to_29_years)/occupational_level_of_young_people_aged_25_to_29_years/3.0/IN33-3.0-occupational-level-of-young-people-aged-25-to-29-years--by-moving-average--table-format.csv

```diff
@@ -0,0 +1,11 @@
+"Year","Lower working background","","","","","Higher working background","","","","","Intermediate background","","","","","Lower professional background","","","","","Higher professional background","","","","","Total background","","","",""
```

Comment:
The last column here is currently called `Total background`.
Perhaps `All backgrounds`, or perhaps we could just omit `background`

Reply:
changed to "Total"